buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -363,6 +363,8 @@ class VagrantTestPlugin implements Plugin<Project> {
     }
 
     private static void createVagrantBoxesTasks(Project project) {
+        File batsDir = new File("${project.buildDir}/${BATS}")
```

Comment:
That we have a constant with the name `BATS` and the value `bats` seems like overkill to me. I probably made it myself though.